app/Jobs/ElasticSearchIndexInit.php

```diff
@@ -52,15 +85,46 @@ public function handle()
                 new \RuntimeException('wbstackElasticSearchInit call for '.$this->wikiDomain.'. No wbstackElasticSearchInit key in response: '.$rawResponse)
             );
 
-            return; //safegaurd
+            return;
         }
 
         if ($response['wbstackElasticSearchInit']['success'] == 0) {
             $this->fail(
                 new \RuntimeException('wbstackElasticSearchInit call for '.$this->wikiDomain.' was not successful:'.$rawResponse)
             );
 
-            return; //safegaurd
+            return;
         }
+
+        $output = $response['wbstackElasticSearchInit']['output'];
+
+        $newlyCreated = "\tCreating index...ok"; // occurs a couple of times when newly created
+        $updated = "\t\tValidating {$this->wikiDomain}_general alias...ok"; // occurs on a successful update run
```

Comment:
This should probably be the wiki db name, or the wiki id.

Reply:
done